index.html

```diff
@@ -237,58 +237,114 @@ <h2 id="conformance-classes">Conformance Classes</h2>
+
+              <dt>CBOR Object Signing and Encryption (COSE):</dt>
+              <dd>COSE secures CBOR (Concise Binary Object Representation) data
+                  structures. CBOR is a binary data format that is more compact than
+                  JSON and is designed for constrained environments.</dd>
+          </dl>
+
+          <p>In the context of Verifiable Credentials:</p>
+          <ul>
+              <li>When using JWTs, the Verifiable Credential or Presentation is
+                  encoded as a JSON Web Token Claim Set.</li>
+              <li>When using SD-JWTs, the Verifiable Credential or Presentation
+                  is encoded as a JSON Claim Set with Selective Disclosure features.</li>
+              <li>When using COSE, the Verifiable Credential or Presentation is
+                  encoded as a CBOR data structure.</li>
+          </ul>
+          <p>
+              In all cases, the underlying data model of the Verifiable Credential
+              or Presentation remains consistent with the [[VC-DATA-MODEL-2.0]],
+              but the encoding and security mechanisms differ.</p>
+          <p>
+              The normative statements in <a data-cite="VC-DATA-MODEL-2.0#securing-mechanisms">
+              Securing Mechanisms</a> apply to securing
+              <code>application/vc-ld+jwt</code> and
+              <code>application/vp-ld+jwt</code>,
+              <code>application/vc-ld+sd-jwt</code> and
+              <code>application/vp-ld+sd-jwt</code>,
+              <code>application/vc-ld+cose</code> and
+              <code>application/vp-ld+cose</code>.
+          </p>
+          <section>
+              <h2>JWT Format and Requirements</h2>
+              <p>
+                  JSON Web Token implementers are advised to review <a data-cite="RFC7519#section-8">Implementation
+                  Requirements</a>.
+              </p>
+              <p>
+                  Issuers, Holders, and Verifiers of JWTs MUST understand the
+                  effect of the JSON Web Token header parameter setting of
+                  <code>"alg": "none"</code> when using JSON Web Tokens to
+                  secure [[[VC-DATA-MODEL-2.0]]] [[VC-DATA-MODEL-2.0]]. When
+                  content types from the [[[VC-DATA-MODEL-2.0]]]
+                  [[VC-DATA-MODEL-2.0]] are secured using JSON Web Tokens, the
+                  header parameter setting of <code>"alg": "none"</code>
+                  is used to communicate that a Verifiable Credential or
+                  Verifiable Presentation encoded as a JWT Claims Set has no
+                  integrity protection.
+              </p>
+              <p>
+                  Issuers, Holders, and Verifiers MUST ignore all JWT Claim Sets that
+                  have no integrity protection.
+              </p>
+              <p>
+                  The JWT Claim Names <code>vc</code> and <code>vp</code>
+                  MUST NOT be present in any JWT Claim Set.
```

Comment:
Is this accurate? We're restricting use of JWT Claim Names `vc` and `vp` in *any* JWT Claim Set, even a JWT Claim Set that does not encode a Verifiable Credential or Verifiable Presentation?